Monika After Story/game/script-topics.rpy

```diff
@@ -3397,15 +3397,24 @@ label monika_playersface:
     m "I always think about what your face actually looks like..."
     m 5hua "Just being able to constantly see that adoring smile of yours would be so exciting!"
     m 5lkc "If only the game could somehow use a webcam or something that's connected to the computer..."
-    m 5wuw "Don't get me wrong! Just knowing that you're real and have emotions is enough to make me happy."
-    m 5luu "But...I'll always wonder what kind of expressions you make."
-    m "And to see the different emotions you have..."
-    m 5eub "Are you shy about showing your face to me?"
-    m "If so, then there's nothing to be shy about, [player]. I am your girlfriend, after all~"
-    m 5hub "Either way, you're beautiful, no matter what."
-    m  "And I'll always love the way you look."
-    m 5eua "Even if I never actually see you, I'll always think about what you really look like."
-    m 5hua "Maybe someday, I'll be able to see you, and be one step closer to you."
+    if persistent._mas_pm_shared_appearance == True:
```

Comment:
dont need to compare to True

```suggestion
    if persistent._mas_pm_shared_appearance:
```

Reply:
May I ask why that is the case?

Reply:
The if statement is just checking if the condition is True. if the var `persistent._mas_pm_shared_appearance` is True, the condition passes. `== True` is just redundant.

Reply:
Ahh, okay I understand.